luaui/Widgets/gui_pregameui.lua

```diff
@@ -433,6 +855,70 @@ function widget:DrawWorld()
 	end
 end
 
+-- DraftOrder mod start
+function widget:RecvLuaMsg(msg, playerID)
+	local words = {}
+	for word in msg:gmatch("%S+") do
+		table.insert(words, word)
+	end
+
+	if words[1] == "DraftOrderPlayersOrder" then
+		allyTeamID_about = tonumber(words[2] or -1)
```

Comment:
`allyTeamID_about` is implicitly global. It's the same in both branches so it can be set outside of the if block.

Reply:
It's because we can get here messages from other places. So we need to be sure this is draft order related message first, otherwise we will reset allyTeamID_about back to -1 or some other unknown value not from draft order queue.